geopandas/io/arrow.py

```diff
@@ -388,9 +402,16 @@ def _read_parquet(path, columns=None, **kwargs):
     parquet = import_optional_dependency(
         "pyarrow.parquet", extra="pyarrow is required for Parquet support."
     )
+    filesystem = kwargs.get("filesystem")
+
+    if _is_fsspec_url(path) and filesystem is None:
+        fsspec = import_optional_dependency(
+            "fsspec", extra="fsspec is requred for 'storage_options'."
+        )
+        filesystem, path = fsspec.core.url_to_fs(path, **(storage_options or {}))
```

Comment:
`url_to_fs` is meant as "public" utility? (`get_fs_token_paths` seems to do somewhat similar things (although `url_to_fs` is certainly more exact what we need), and is listed in the API docs (https://filesystem-spec.readthedocs.io/en/latest/api.html#fsspec.core.get_fs_token_paths), while `url_to_fs` is not listed there)

Reply:
https://github.com/intake/filesystem_spec/issues/752

Reply:
Thanks!